thumbor/src/thumbor_/__main__.py

```diff
@@ -1,12 +1,34 @@
+import asyncio
+import sys
 from argparse import ArgumentParser
 
-from . import __version__
+from thumbor_.server import main as server_main
```

Comment:
How much work would it take to move to `from thumbor.server import main as server_main`? We then wouldn't need our own `server.py`

Reply:
The thing is we have added the upload handler to the server so Im not sure what the best approach to do this would be?